site/docs/webrtc/overview.md

```diff
@@ -66,44 +51,42 @@ More can be found on this in our [quickstart guide](/docs/webrtc/quickstart).
 
-There is a limit of 20 participants in a session. This includes all participants, whether they are Audio-only, Audio & Video, or dialed-in participants.
+## Limitations
 
-### Session Tags
+There is a limit of 20 participants in a session.
 
-You can add a tag to a session to help you identify them with friendly names. Session Tag information is passed on to your Billing Reports, which can facilitate billing of clients if you do pass-through billing or auditing. These can also be used to allow for easier understanding or grouping of sessions.
+## Participant-Session Membership
 
-Example uses of a Participant Tags:
+A participant by itself cannot receive media. It must first be associated with, or a member of, a session. Only then can it receive media from other session participants. The (Add Participant to Session Endpoint)[https://new.dev.bandwidth.com/apis/webrtc#operation/addParticipantToSession] is used to add a participant to a session. The target participant and session are indicated by ID in the URL path:
 
-- Meeting IDs to make it easier to associate a session with internal scheduling or management
-- Externally safe Account UUIDs to assist with allocating costs from a Billing Report.
+```
+/sessions/{sessionId}/participants/{participantId}
+```
 
-#### Restrictions/Limitations
+After adding a participant to the session a participant’s subscription information can then be updated, which describes which media streams a participant will receive. Information about participant subscriptions can be optionally supplied in the request body for the (Add Participant to Session Endpoint)[https://new.dev.bandwidth.com/apis/webrtc#operation/addParticipantToSession]. Subscriptions can also be updated via a call to a separate endpoint for updating subscriptions, the (Update Participant Subscriptions Endpoint)[https://new.dev.bandwidth.com/apis/webrtc#operation/updateParticipantSubscriptions]. Subscriptions are described in further detail below.
 
-- Tags MUST NOT include Personally Identifying Information (PII). We recommend using UUIDs if you need to reference customers or users.
-- No uniqueness is enforced by the WebRTC system for tags. Duplicates are acceptable.
-- A Session tag is limited to 50 characters
 
 ## Subscriptions
 
-When a Participant receives the audio or video Streams from another Participant, it is via a Subscription. Subscriptions are very flexible, allowing for the creation of custom scenarios where participants may hear or see some people and devices, but not others. Subscriptions can be created three levels: \
+When a Participant receives the audio or video Streams from another Participant, it is via a Subscription. A subscription describes which media streams a participant will receive. Note that a participant must be a member, or part of, a Session before it can receive any media from other participants.
```

Comment:
make Streams lowercase

Reply:
We should be consistent throughout with out capitalization of terms like Participant, Subscription, etc. It looks like in this doc we lower case them